### PR TITLE
Fix Node.Js issues

### PR DESCRIPTION
**Pull request by blu3id**
node_modules should be local/platform specific, this removes them from repo and adds appropriate .gitignore and some set-up instructions into readme

### CODE REVIEW

**Review by barneycarroll — APPROVED** [review]
Good move. Bit risky but I think I'd rather clone the repo after this is merged in!

**Comment by DeckOfPandas** [comment]
@barneycarroll Thanks. You should have write access, so do whatever you think.

**Comment by DeckOfPandas** [comment]
I can't build now:
```
:nhsideaslab.com helen$ npm install gulp -cli -g
npm WARN Paper@1.7.1 No repository field.
```
Next:
```
:nhsideaslab.com helen$ gulp watch
module.js:471
    throw err;
    ^

Error: Cannot find module 'fs-extra'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/helen/Files/ohw/nhs-ideas-lab/nhsideaslab.com/gulpfile.js:2:10)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Pull request merged**
